src/BTVNanoCommissioning/helpers/definitions.py

```diff
@@ -0,0 +1,505 @@
+def definitions(defaults=True):
```

Comment:
any chance all this info could be reformated as a dictionary like 
`{name: {'range': [], 'display_name': "foo", 'bins': N, 'format': ".2f"}}`

a set of them for each property

Reply:
Done